sci-libs/openblas/openblas-0.3.6.ebuild

```diff
@@ -43,11 +43,17 @@ src_unpack () {
 	default
 	find "${WORKDIR}" -maxdepth 1 -type d -name \*OpenBLAS\* && \
 		mv "${WORKDIR}"/*OpenBLAS* "${S}" || die
+	if use index-64bit; then
+		cp -aL "${S}" "${S}-index-64bit"
```

Comment:
Add `|| die` after `cp`.  Portage external commands need to be guarded against failure.